src/main/java/frc/robot/Robot.java

```diff
@@ -32,10 +35,15 @@ public class Robot {
 	public static final RobotType ROBOT_TYPE = RobotType.determineRobotType();
 
 	private final Swerve swerve;
+
 	private final ElevatorRoller elevatorRoller;
+
 	private final Funnel funnel;
+
 	private final IntakeRoller intakeRoller;
 
+	private final Flywheel flywheel;
```

Comment:
remove lines between
```suggestion
	private final ElevatorRoller elevatorRoller;
	private final Funnel funnel;
	private final IntakeRoller intakeRoller;
	private final Flywheel flywheel;
```